src/backend/tools.ts

```diff
@@ -193,25 +196,29 @@ export const DXVK = {
 
 export const Winetricks = {
   download: async () => {
+    if (!isLinux) {
+      return
+    }
+
     const url =
       'https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks'
     const path = `${heroicToolsPath}/winetricks`
-    const downloadCommand = `curl -L ${url} -o ${path} --create-dirs`
 
     if (!isOnline()) {
       return
     }
 
-    return execAsync(downloadCommand)
-      .then(() => {
-        exec(`chmod +x ${path}`)
-        logInfo('Downloaded Winetricks', { prefix: LogPrefix.WineTricks })
-      })
-      .catch(() => {
-        logWarning('Error Downloading Winetricks', {
-          prefix: LogPrefix.WineTricks
-        })
+    try {
+      logInfo('Downloaded Winetricks', { prefix: LogPrefix.WineTricks })
```

Comment:
This would be
```suggestion
      logInfo('Downloading Winetricks', { prefix: LogPrefix.WineTricks })
```
now, right?

Reply:
fixed